### ISSUE TITLE
[Court] Unclear on what I should do in a closed case at 100%

### ISSUE THREAD

**Issue by maltekliemann** [zeitgeistpm/ui#2067]
"Waiting to be reassigned" - Should I be doing something here? Are the stakes reassigned automatically? Seems unclear. The case ended a month ago.

<img width="1920" alt="Screenshot 2023-12-08 at 11 45 24" src="https://github.com/zeitgeistpm/ui/assets/49552287/328636ec-f386-4b84-af90-22eeecf2ade5">

**Comment by yornaath** [comment]
Yeah its unclear since the ended date (2 nov 2023) is the market end date and not the case end date. I think we also need indexing to know when a case was settled @saboonikhil
